### PR TITLE
More tests for whitespace in multi-line strings

### PR DESCRIPTION
**Pull request by tjol**
@tabatkins please check that I got these right.

### CODE REVIEW

File: tests/test_cases/expected_kdl/slashdash_newline_before_node.kdl [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (tabatkins) [inline tests/test_cases/expected_kdl/slashdash_newline_before_node.kdl:1]: Looks like you accidentally included an old version of this expectation; current testsuite has a newline in it.

Comment (tjol) [reply tests/test_cases/expected_kdl/slashdash_newline_before_node.kdl:1]: Ah, I somehow thought the file had two newlines (and removed the second one).

Comment (tjol) [reply tests/test_cases/expected_kdl/slashdash_newline_before_node.kdl]: Oooh, the file contains CR LF, not just LF. (That's why my test is failing)

That can't be on purpose, can it?

**Review by tabatkins — APPROVED** [review]
